packages/flutter_tools/lib/runner.dart

```diff
@@ -97,18 +96,6 @@ Future<int> run(
           globals.printStatus('Analytics reporting enabled.');
         }
 
-        // Send an event to GA3 for any users that are opted into GA3
-        // analytics but have opted out of GA4 (package:unified_analytics)
-        // TODO(eliasyishak): remove once GA3 sunset, https://github.com/flutter/flutter/issues/128251
-        if (!globals.analytics.telemetryEnabled &&
-            globals.flutterUsage.enabled) {
-          UsageEvent(
-            'ga4_and_ga3_status_mismatch',
-            'opted_out_of_ga4',
-            flutterUsage: globals.flutterUsage,
-          ).send();
-        }
```

Comment:
IDK if anyone ever looked for these events, but they are sure gone by now if they ever existed 🤷